.github/workflows/email.yml

```diff
@@ -42,7 +42,16 @@ jobs:
         id: changed-files
         uses: tj-actions/changed-files@v34
         with:
-          separator: ","                 
+          separator: "," 
+      # tj-actions' limiation prevents the user from using "\n" as ouptput seperator.
```

Comment:
"limitation" and "output" are misspelled

Reply:
Fixed